src/providers/wms/qgstilescalewidget.cpp

```diff
@@ -102,11 +103,18 @@ void QgsTileScaleWidget::scaleChanged( double scale )
 
 void QgsTileScaleWidget::mSlider_valueChanged( int value )
 {
-  Q_UNUSED( value );
   QgsDebugMsg( QString( "slider released at %1: %2" ).arg( mSlider->value() ).arg( mResolutions.at( mSlider->value() ) ) );
+
+  // Invert value in tooltip to match expectation (i.e. 0 = zoomed out, maximum = zoomed in)
+  QToolTip::showText( QCursor::pos(), QString( "Zoom level: %1\nResolution: %2" ).arg( mSlider->maximum() - value ).arg( mResolutions.at( value ) ), this );
```

Comment:
`tr(…)`

Reply:
@jef-n , is it worth breaking the string into two translatable parts, i.e.:  `tr( "Zoom level: %1" ).arg( ... ) + "\n" + tr( "Resolution: %2" ).arg( ... )` ?